src/Nix/Expr/Types.hs

```diff
@@ -483,7 +569,7 @@ $(makeTraversals ''NKeyName)
 $(makeTraversals ''NUnaryOp)
 $(makeTraversals ''NBinaryOp)
 
--- $(makeLenses ''Fix)
+--x $(makeLenses ''Fix)
```

Comment:
I guess these are to avoid Haddock warnings, right?

Reply:
I believe haddock parses this as a "named chunk", so in some way these comments even end up in the rendered documentation, for example here:

http://hackage.haskell.org/package/hnix-0.9.0/docs/Nix-Expr-Types.html#v:_NEq

Reply:
I've reported this in https://github.com/haskell/haddock/issues/1210.